### ISSUE TITLE
WARNING INFO! The text type in bs-input component is not working.

### ISSUE THREAD

**Issue by xereda** [yuche/vue-strap#347]
The text type in bs-input component is not working.

The component is not allowing you to enter words with space.

The text type in bs-input component is not working.

The component is not allowing you to enter words with space.

We await an immediate return.

**Comment by santigarcor** [comment]
@xereda The example in the docs page is applying a `:mask` to that text input:

```html
    <bs-input :value.sync="input" label="Username" help="Only allows lowercase letters and numbers." error="Insert username" placeholder="Username can't start with a number." pattern="^[a-z][a-z0-9]+$" :mask="mask" minlength="5" readonly required icon></bs-input>
```

So there's nothing wrong with it.

**Comment by xereda** [comment]
Look this: https://jsbin.com/laxexukobu/1/edit?html,output

**Comment by santigarcor** [comment]
@xereda You're right i just updated to v1.1.23 and i can't type spaces and i'm getting this error:

`[Vue warn]: Error when evaluating expression "function nativeValidate() {
	      return this.$els.input.checkValidity && (~['url', 'email'].indexOf(this.type.toLowerCase()) || this.min || this.max);
	    }": TypeError: Cannot read property 'checkValidity' of undefined (found in component: <bs-input>)`

**Comment by santigarcor** [comment]
In version 1.1.22 the problem isn't there

**Comment by xereda** [comment]
Vue-Strap Versão 1.1.22 (this ok): https://jsbin.com/jesocixufi/1/edit?

Vue-Strap Versão 1.1.23 (this version have a bug): https://jsbin.com/mepagaxaya/1/edit?html,output

**Comment by metamet** [comment]
Can confirm the bug exists in 1.1.23. All input forms are removing spaces as well. 

`bundle.js:5882 [Vue warn]: Error when evaluating expression "function (){return this.$els.input.checkValidity&&(~["url","email"].indexOf(this.type.toLowerCase())||this.min||this.max)}": TypeError: Cannot read property 'checkValidity' of undefined (found in component: <bs-input>)`

**Comment by wffranco** [comment]
You're right, this week I was sick so I don't touch any code.
I have some pending changes on Input, if I can I will have all ready tomorrow.
That's an important bug so I'll fix it even if I'm not ready with the other fixes.

Greeting.

**Comment by wffranco** [comment]
Fixed in 1.1.24. Sorry for the delay, I was sick.
